src/components/incidents/Incidents.js

```diff
@@ -265,95 +239,22 @@ const Incidents = () => {
-      option.value.slice(0, inputValue.length).toLowerCase()
-      ? option
-      : null;
+  const onFilterChange = (key, val) => {
+    // TODO make filtering by city work, ideally by changing the filtering system to
+    // work on the entire form data instead of one value at a time
+    const changeFns = {
+      force_rank: onRank,
+      state: setUsState,
+      tags: setActiveCategories,
+      date_range: onDateSelection,
+    };
+    changeFns[key](val);
   };
 
   return (
     <div className="incident-reports-page">
       <div className="form-container">
-        <h1>Filter Incident Reports</h1>
-        <form className="export-form">
-          <label htmlFor="ranks" className="labels">
-            Rank
-            <br></br>
-            <Select
-              onChange={onRank}
-              showSearch
-              defaultValue="All"
-              className="form-inputs"
-              id="ranks"
-              value={rank}
-            >
-              <Option value="All">All</Option>
-              <Option value="1">Rank: 1</Option>
-              <Option value="2">Rank: 2</Option>
-              <Option value="3">Rank: 3</Option>
-              <Option value="4">Rank: 4</Option>
-              <Option value="5">Rank: 5</Option>
-            </Select>
-          </label>
-          <label htmlFor="locations" className="labels">
-            Location
-            <br></br>
-            <SearchBar className="form-inputs" setUsState={setUsState} />{' '}
-          </label>
-          <label htmlFor="categories" className="labels">
-            Category
-            <br></br>
-            <AutoComplete
-              value={value}
-              options={categoriesData}
-              onSelect={onCategorySelect}
-              onChange={onCategoryChange}
-              allowClear={true}
-              filterOption={filterOption}
-              placeholder="Browse Categories"
-              notFoundContent="Category Not Found"
-              id="categories"
-              className="form-inputs"
-            />
-            {activeCategories &&
-              activeCategories.map(tag => {
-                return (
-                  <CheckableTag
-                    style={{
-                      marginTop: 3,
-                      backgroundColor: '#003767',
-                    }}
-                    key={tag}
-                    checked={activeCategories.indexOf(tag) > -1}
-                    onChange={checked => onToggle(tag, checked)}
-                  >
-                    {tag}
-                  </CheckableTag>
-                );
-              })}
-          </label>
-          <label htmlFor="dates" className="labels">
-            Date
-            <br></br>
-            <RangePicker
-              onCalendarChange={onDateSelection}
-              className="form-inputs"
-            />
-          </label>
-        </form>
+        <FilterForm onValuesChange={onFilterChange} />
```

Comment:
Thanks for working on this file. It's the longest/messiest file in the whole project.